InvenTree/part/test_part.py

```diff
@@ -616,6 +616,8 @@ def _notification_run(self, run_class=None):
         # reload notification methods
         storage.collect(run_class)
 
+        NotificationEntry.objects.all().delete()
```

Comment:
Simple but effective 👍

Reply:
Not actually all that effective though... These notification unit tests are really frustrating!